packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -34,9 +34,12 @@
   background-color: var(--color-surface--background);
 }
 
+.option:focus {
+  background-color: rgb(34, 153, 221);
```

Comment:
reminder: delete this before merging. only have it so that I can easily see the focus moving around.

Reply:
DELETED